usr/share/rear/backup/NETFS/default/100_mount_NETFS_path.sh

```diff
@@ -2,4 +2,4 @@ if [[ "$BACKUP_MOUNTCMD" ]] ; then
     BACKUP_URL="var://BACKUP_MOUNTCMD"
 fi
 
-mount_url $BACKUP_URL $BUILD_DIR/outputfs $BACKUP_OPTIONS
+mount_url "$BACKUP_URL" $BUILD_DIR/outputfs $BACKUP_OPTIONS
```

Comment:
`$BUILD_DIR/outputfs` should be quoted as well for consistency with other changes